files/manifests/k8s-api-server.yaml

```diff
@@ -26,9 +26,9 @@ spec:
     {{ range .Kubernetes.Apiserver.CommandExtraArgs -}}
     - {{ . }}
     {{ end -}}
-    {{- if .Etcd.HighAvailability }}
+    {{- if .Etcd.HighAvailability -}}
```

Comment:
This doesn't collapse too much whitespace (e.g. putting multiple flags on one line) when used with or without `.Kubernetes.Apiserver.CommandExtraArgs`?

Reply:
I think it's fine. Take a look at the updated example https://play.golang.org/p/DxKxmnrURtG

Reply:
Cool, thanks.